src/test/scala/latis/server/CapabilitiesServiceSpec.scala

```diff
@@ -0,0 +1,37 @@
+package latis.server
+
+import cats.effect.IO
+import io.circe._
+import io.circe.syntax._
+import org.http4s._
+import org.http4s.implicits._
+import org.scalatest.FlatSpec
+
+class CapabilitiesServiceSpec extends FlatSpec {
+
+  "The capabilities service" should "advertise all output options" in {
```

Comment:
ScalaTest leans towards BDD-style tests, which is different from the JUnit-style tests we've been writing.

I like how this forces me to write out what ought to be happening, which gives more structure and direction to the test because I need to be able to describe it succinctly. However, I can also see this being difficult.

ScalaTest lets you combine multiple styles of testing, so maybe this isn't an issue. There could be specification-type tests and JUnit-type tests for things like reproducing a particular bug.

Reply:
I use snake_case for test names so I can be more verbose, a recommendation I picked up somewhere. I can see this becoming a bit onerous though it makes things more clear. I can always cut and paste.

The one think I want back is the ability to run a single test (not an entire class) in eclipse. I lost that with JUnit after a recent upgrade.